meta/main.yml

```diff
@@ -5,14 +5,14 @@ galaxy_info:
   description: An Ansible role for installing Open Trip Planner.
   company: Azavea Inc.
   license: Apache
-  min_ansible_version: 1.5
+  min_ansible_version: 2.0
   platforms:
   - name: Ubuntu
     versions:
-    - trusty
+    - xenial
```

Comment:
The `ansible.java` role has support for Oracle Java 8 (not specifically OpenJDK 8) on Trusty. The OpenTripPlanner docs seem to claim that either flavor is acceptable:

> As a Java program OTP must be run under a Java virtual machine (JVM), which is provided as part of the Java runtime (JRE) or Java development kit (JDK). Run java -version to check that you have version 1.8 or newer of the JVM installed. If you do not you will need to install a recent OpenJDK or Oracle Java package for your operating system.